aspnetcore/includes/net-core-prereqs-vs-3.0.md

```diff
@@ -0,0 +1,2 @@
+* [Visual Studio 2017 version 15.9 or later](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload
```

Comment:
Use the campaign link to allow tracking:

```suggestion
* [Visual Studio 2017 version 15.9 or later](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the **ASP.NET and web development** workload
```

Also, I suspect VS 2019 will be required when 3.0 releases. Do we have an issue to update prereqs accordingly?

Reply:
>Also, I suspect VS 2019 will be required when 3.0 releases. Do we have an issue to update prereqs accordingly?

I don't think we have been creating a dedicated issue for updating prereq includes each release. I would wait until VS 2019 becomes required; that'll be the sort of change that we won't need a reminder to change prereqs for.